client/my-sites/plan-features/header.jsx

```diff
@@ -3,12 +3,14 @@
  */
 import React, { Component, PropTypes } from 'react';
 import noop from 'lodash/noop';
+import identity from 'lodash/identity';
```

Comment:
This is another leftover piece of the enzyme tests. We test the bare component (without localize), so a natural fallback is identity to parrot any text fields.